src/customize-pop/application-notifications.md

```diff
@@ -0,0 +1,31 @@
+# Application Notifications
+
+Settings to control application notification behavior are located in the `Settings` ➞ `Notifications` tab.
+
+![Notifications Settings](/images/application-settings/notifications-settings.png)
+
+| Option | Function |
+|--------|----------|
+| Do Not Disturb | Disable all application notification pop-ups globally. Notifications will still queue up in the notiications drop-down while this setting is enabled. |
+| Lock Screen Notifications | Enable setting toggling for the `Lock Screen Notifications` and `Show Message Content on Lock Screen` options. |
```

Comment:
> Enable setting toggling for the `Lock Screen Notifications` and `Show Message Content on Lock Screen` options.

This does not read clearly. I would say something simpler like:

> Enable or disable showing notifications on the lock screen. If enabled, this can be further configured per-application.

Reply:
Added your suggested wording